app/views/contact/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<% content_for :title, "#{t('.title')} - GOV.UK" %>
```

Comment:
I assume this automatically gets this from the contact: part of the locale because it knows about the controller - which is nice, but I think they're explicit in the existing views in frontend, so maybe we should be the same (ie t("contact.title"))